zipline-bytecode/src/test/kotlin/app/cash/zipline/bytecode/EncodingTest.kt

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.zipline.bytecode
+
+import com.google.common.truth.Truth.assertThat
+import okio.Buffer
+import org.junit.Test
+
+class EncodingTest {
+  @Test fun intGetBit() {
+    assertThat(0b00000001.bit(0)).isFalse()
```

Comment:
A bit confusing that `bit`'s parameter is zero-based, should it be one-based, since you need "the first bit"?

Reply:
No? The first element in a list is also 0-based, right?

Reply:
It behaves like an array. The first element is the 0th.